app/config/config.yml

```diff
@@ -222,51 +222,8 @@ sonata_doctrine_phpcr_admin:
     templates:
         form:
             - admin/form_admin_fields.html.twig
-    document_tree_defaults: [locale]
     document_tree:
-        Doctrine\ODM\PHPCR\Document\Generic:
-            valid_children:
-                - all
```

Comment:
I think so. The tree itself, as code construct, currently has got no restrictions anymore. But i think we should introduce a kind of an enhancer which add the doctrine-phpcr information into the response and force the tree-browser to use that one. And yes i think we need those restrictions at least when restrict the targets on which we are able to move. But that is no feature of fancytree as far as i know  – we have to implement that on our own js code.